app/models/deployment.rb

```diff
@@ -0,0 +1,177 @@
+class Deployment < ApplicationRecord
```

Comment:
`Deployment` is a very generic name.  Can this be pseudo namespaced to `ContainerDeployment`?  Or, is there another way to make this more specific?